x-pack/plugins/apm/public/components/app/RumDashboard/RumHome.tsx

```diff
@@ -5,33 +5,63 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useContext } from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiFlexGroup, EuiTitle, EuiFlexItem } from '@elastic/eui';
 import { RumOverview } from '../RumDashboard';
-import { CsmSharedContextProvider } from './CsmSharedContext';
+import { CsmSharedContext, CsmSharedContextProvider } from './CsmSharedContext';
 import { WebApplicationSelect } from './Panels/WebApplicationSelect';
 import { DatePicker } from '../../shared/DatePicker';
 import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 import { UxEnvironmentFilter } from '../../shared/EnvironmentFilter';
 import { UserPercentile } from './UserPercentile';
 import { useBreakPoints } from '../../../hooks/use_break_points';
+import { KibanaPageTemplateProps } from '../../../../../../../src/plugins/kibana_react/public';
 
 export const UX_LABEL = i18n.translate('xpack.apm.ux.title', {
   defaultMessage: 'Dashboard',
 });
 
 export function RumHome() {
-  const { observability } = useApmPluginContext();
+  const { core, observability } = useApmPluginContext();
   const PageTemplateComponent = observability.navigation.PageTemplate;
 
   const { isSmall, isXXL } = useBreakPoints();
 
   const envStyle = isSmall ? {} : { maxWidth: 500 };
 
+  // TODO: NOT THE RIGHT METRIC TO CHECK
```

Comment:
are you still planing to make this PR work? if yes i can help with this metric check,

Reply:
@shahzad31 Yes, this PR is slated for 7.16 and anywhere there are "TODO" or "HELP" comments, are parts that I need Obs folks to jump in on. So all the help is gratefully needed!

Reply:
Added a PR for UX app has no data screen https://github.com/elastic/kibana/pull/111866